prime-router/docs/proposals/0009-internal-tools-migration-to-superset.md

```diff
@@ -0,0 +1,50 @@
+## Background
+
+The PRIME ReportStream uses Metabase for business analytics. Metabase is a business intelligence tool that allows for the exploration, investigation, and visualization of data. With this internal tool, our team is able to generate charts and dashboards from our stored data. Even though this instrument provides quality charts, it also has its shortcomings:
+
+### Limitations 
+
+- Query complexity 
+    - Limited to simple queries only
+    
+- Data Source 
+    - Only works well with a single SQL data source
+    - Does not allow for joining of data
+    - Unable to upload CSV file format
+    
+- Formatting
+    - Can be difficult to customize charts
+    
+
+As our data input becomes larger and keeping scalability in mind, these limitations will become problematic. It will be more time-consuming to write complex queries and to customize charts than it is now.
+
+## Goals
+
+Our goal is to meet the demands of scalability while maintaining the ability to analyze data by dicing and visualization of charts and reports. Other key factors to consider are:
```

Comment:
Metabase can be slow, but the initial culprit is likely that's because our database isn't fully tuned.

Reply:
Out of interest, what is the data volume in metabase at this point in time?